simple_toml_settings/settings.py

```diff
@@ -71,8 +74,13 @@ def get_settings_folder(self) -> Path:
             return Path.home()
 
         settings_folder: Path = Path.home() / f".{self.app_name}"
+
+        if self.xdg_config:
+            settings_folder = (
+                Path.home() / xdg_config_home() / f"{self.app_name}"
```

Comment:
adding the `Path.home()` here is not needed, since `xdg_config_home()` already does this. Python `Path` will sort the duplication out but could create issues if we change stuff
```suggestion
                xdg_config_home() / f"{self.app_name}"
```